infer/lib/rust/tests/programs/operands/move/move_from_tuple.sil

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) Facebook, Inc. and its affiliates. 
+//
+// This source code is licensed under the MIT license found in the 
+// LICENSE file in the root directory of this source tree. 
+
+.source_language = "rust"
+
+define test() : void {
+  local var_0: void, var_1: (string, int), var_2: string, var_3: string
+
+  #node_0:
+    %0 = String::from("hello")
+    store &var_2 <- %0
+
+    %1 = (var_2, 42)
+    store &var_1 <- %1
+
+    %2 = var_1.0
+    store &var_3 <- %2
+
+    call drop_in_place(&var_3)
```

Comment:
This is not valid textual, it should be: (I will update this in the translation rules)
```
        n0 = load &var_3
        n1 = __sil_free(n0)
```